.github/libs/versionUpdater.js

```diff
@@ -0,0 +1,69 @@
+const utils = require('util');
+const exec = utils.promisify(require('child_process').exec);
+
+const semanticVersionLevels = {
+    major: 'MAJOR',
+    minor: 'MINOR',
+    patch: 'PATCH',
+    build: 'BUILD',
+};
+const maxIncrements = 999;
+
+const getVersionNumberFromString = (versionString) => {
+    const [version, build] = versionString.slice(1).split('-');
+    const [major, minor, patch] = version.split('.').map(n => Number(n));
+    return [major, minor, patch, build ? Number(build) : undefined];
+};
+
+const getVersionStringFromNumber = (major, minor, patch, build) => {
```

Comment:
```suggestion
const getVersionStringFromNumber = (major, minor, patch, build = null) => {
```

Just to make it clear that `build` is optional. Also should be reflected in the `JSDoc` like this:

```javascript
/**
 * ...
 * @param {Number} patch
 * @param {Number} [build] - the square brackets mark it as optional
 * @returns {String}
 **/
```